### PR TITLE
[SpikeSynchrony] Update URL

### PR DESCRIPTION
**Pull request by russelljjarvis**
Updating links with correct information, which reflect the intended custodianship of the source code.

### CODE REVIEW

File: S/SpikeSynchrony/Package.toml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 name = "SpikeSynchrony"
 uuid = "3713708d-6816-4a57-846e-904b81e02fa1"
-repo = "https://github.com/russelljjarvis/SpikeSynchrony.jl.git"
+repo = "https://github.com/JuliaNeuroscience/SpikeSynchrony.jl"
```
Comment (Datseris) [inline S/SpikeSynchrony/Package.toml:3]: You still need the `.git` at the end. It's suppsed to point to a git repository.

Comment (russelljjarvis) [reply S/SpikeSynchrony/Package.toml:3]: Okay let me retry.

**Review by giordano — CHANGES_REQUESTED** [review]
This can't work. The new repository is behind the original one, the registered versions will become uninstallable

**Comment by Datseris** [comment]
@russelljjarvis make a pull request to the original repo that updates it to your fork

**Comment by russelljjarvis** [comment]
Yes, AOK, just woke up. Will do.

**Comment by russelljjarvis** [comment]
Done.

**Comment by Datseris** [comment]
@giordano we have a problem, but I don't know how to fix... In a severe case of misscommunication, the user has done _different_ commits at the forked repo and different commits at the original repo (whose link we try to establish here), and now their git history is totally out of sync. Do you have any suggestions?

**Comment by russelljjarvis** [comment]
Hi @giordano, I wonder if its possible to delete the original registration of SpikeSynchrony.jl that I made, it was a big mistake @Datseris has been working with me to fix the situation, but I worry that I am wasting everyone's time and it might be faster to delete the commit to the registry?

**Comment by giordano** [comment]
>  I wonder if its possible to delete the original registration of SpikeSynchrony.jl that I made

[No](https://github.com/JuliaRegistries/General/blob/206e49368ea9689808bdbc60a86525f06e572ce9/README.md#how-do-i-remove-a-package-or-version-from-the-registry).

**Comment by giordano** [comment]
But now we can merge this pull request, since the upstream repository is in sync with the fork and the registered git-tree-sha1 is present in the repository:
```console
% git log --pretty=format:'%T %s' | grep bce0c01f61c6b76caa9022196850ec18e38d0e63
bce0c01f61c6b76caa9022196850ec18e38d0e63 Merge pull request #6 from russelljjarvis/master
bce0c01f61c6b76caa9022196850ec18e38d0e63 Update SPIKEdistance.jl
```

If you really want, you can yank the registered version, i.e., add the line `yanked = true` after line
https://github.com/JuliaRegistries/General/blob/f28830bb8faca1ee185e1573f4371b38ac003253/S/SpikeSynchrony/Versions.toml#L2
which is pretty much equivalent to making it inaccessible, without wrecking the registry.

**Pull request merged**